x-pack/solutions/security/plugins/cloud_security_posture/public/pages/vulnerabilities/latest_vulnerabilities_table.tsx

```diff
@@ -80,14 +79,8 @@ const CspVulnerabilityFindingRenderer = ({
   return children({ finding });
 };
 
-const flyoutComponent = (row: DataTableRecord, onCloseFlyout: () => void): JSX.Element => {
-  return (
-    <CspVulnerabilityFindingRenderer row={row}>
-      {({ finding }) => (
-        <VulnerabilityFindingFlyout vulnerabilityRecord={finding} closeFlyout={onCloseFlyout} />
-      )}
-    </CspVulnerabilityFindingRenderer>
-  );
+const onOpenFlyoutCallback = (): JSX.Element => {
+  return <></>;
```

Comment:
Is this needed here?

Reply:
for Vulnerabilities table no (as we don't have any metric to track here) but for Misconfiguration table we do 
but since both of Vuln and Misconfiguration uses UnifiedDataTable that requires this for renderDocumentView I just keep it